pkg/api/v1/types.go

```diff
@@ -175,6 +175,17 @@ type ObjectMeta struct {
 	// queryable and should be preserved when modifying objects.
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/annotations.md
 	Annotations map[string]string `json:"annotations,omitempty"`
+
+	// List of objects depended by this object. If ALL objects in the list have
+	// been deleted, this object will be garbage collected.
+	OwnerReferences []ObjectReference `json:"ownerReferences,omitempty"`
+
+	// Must be empty before the object is deleted from the registry. The map key
+	// is an identifier for the responsible component that will remove the entry
+	// from the map, the value is an ad hoc, explanatory sentence (e.g.,
+	// "orphaning finalizer"). If the ObjectMeta.DeletionTimestamp is non-nil,
```

Comment:
This is user-facing documentation. Don't mention Go types/fields. Use json field names.